server/reload_test.go

```diff
@@ -20,6 +20,7 @@ import (
 	"encoding/json"
 	"flag"
 	"fmt"
+	"github.com/nats-io/jwt/v2"
```

Comment:
Could you move this one with the `github.com/` ones?

Reply:
changed